thrifty-runtime/src/jvmMain/kotlin/com/microsoft/thrifty/transport/TTransportException.kt

```diff
@@ -0,0 +1,6 @@
+package com.microsoft.thrifty.transport
+
+class TTransportException : Throwable {
```

Comment:
I tend to think this exception doesn't buy us anything, and would prefer not to include it.  We could use the perfectly-good `ProtocolException` in its stead.

Reply:
`ProtocolException` unfortunately does not have the possibility to pass a `cause` (doesn't have that constructor implemented), and I really don't like wrapping exceptions in a way that loses information.
But I do agree with you that this does not provide much value, so I will go for just not catching and rethrowing these wrapped.

Reply:
OK, SGTM.